public/javascripts/insert-paragraph.js

```diff
@@ -0,0 +1,13 @@
+
+
+exports.insertParagraph = (paragraph) => {
```

Comment:
I would rename this file to /javascripts/data/paragraph.js. This file can be where all of your database functions related to creating, updating, reading, and deleting paragraphs from the DB will live